homework-submission/homework-week-4/src/components/week3-component/people.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import './people.css';
+import logo from './logo.svg';
+
+// Main Component
+class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      people: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('https://uinames.com/api/?amount=10')
```

Comment:
What if the request fails? Make sure you add a `catch` for the errors.

Reply:
Error check method has been added.